src/main/java/org/spongepowered/api/event/entity/MoveEntityEvent.java

```diff
@@ -70,102 +72,14 @@
      * Fired when an {@link Entity}'s position changes.
      */
     @GenerateFactoryMethod
-    interface Position extends MoveEntityEvent {}
+    interface Position extends MoveEntityEvent {
```

Comment:
This was added to be able to separate position and rotation movement - but the rotation event is actually separate. So why do we need this? It would make more sense if this event contained entity rotation... but it does not.

I think we should specify that the `MoveEntityEvent` refers to position only, and that the rotation event is rotation only and just separate the movement and rotation logic completely.